pkg/chains/signing.go

```diff
@@ -223,10 +234,20 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 	if err := MarkSigned(ctx, tektonObj, o.Pipelineclientset, extraAnnotations); err != nil {
 		return err
 	}
-
+	logger.Debugf("Recording count of marked payload for %v", tektonObj)
```

Comment:
Trace level?

Reply:
Removed the loggers as trace level  is not available.